DigitalOffice/Services/UserService/Validators/UserLoginValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UserService.Models;
+using FluentValidation;
+
+namespace UserService.Validators
+{
+    public class UserLoginValidator : AbstractValidator<UserCredentials>
+    {
+        public UserLoginValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .WithMessage("Email is empty.")
+                .MaximumLength(50)
```

Comment:
A strange limitation.

Reply:
why

Reply:
[link](https://remoteshaman.com/programming/common/rfc-the-best-guide-to-standards#:~:text=%D0%98%D0%B7%20%D0%B4%D0%BE%D0%BA%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%B0%20%22RFC%20821%22%20%D0%BA%D0%BE%D1%82%D0%BE%D1%80%D1%8B%D0%B9,Mail%20%D0%B0%D0%B4%D1%80%D0%B5%D1%81%D0%B0%20%D1%81%D0%BE%D1%81%D1%82%D0%B0%D0%B2%D0%BB%D1%8F%D0%B5%D1%82%20129%20%D1%81%D0%B8%D0%BC%D0%B2%D0%BE%D0%BB%D0%BE%D0%B2.)

Reply:
fixed